external-crates/move/crates/move-compiler/tests/move_2024/parser/invalid_macro_locs.exp

```diff
@@ -91,9 +96,14 @@ error[E01016]: invalid name
 error[E04001]: restricted visibility
    ┌─ tests/move_2024/parser/invalid_macro_locs.move:39:9
    │
+36 │     public struct S<T> { u: T }
+   │                   - Struct may only be constructed in the module in which they are declared
+   ·
 39 │         S{ u: $u }
-   │         ^^^^^^^^^^ Invalid instantiation of '0x42::m::S'.
-All structs can only be constructed in the module in which they are declared
+   │         ^^^^^^^^^^ Invalid instantiation of '0x42::m::S'
```

Comment:
I like this, but maybe we should give some indication of visibility in this error, since you might not be able to see the other error (particularly in the IDE)